### DISCUSSION TITLE
ArgoCD Git repositories polling frequency

### DISCUSSION THREAD

**Discussion by HaniHamdani** [argoproj/argo-cd#18057] [Q&A]
Hello,

I'm inquiring if there's a method to adjust the default polling frequency of Argo CD, which currently stands at 3 minutes, for detecting changes in the manifests within Git repositories. I'm exploring alternative solutions aside from using Webhooks. Any insights or recommendations would be greatly appreciated.

Thank you.

**Comment by ptr1120** [comment]
See in the docs https://argo-cd.readthedocs.io/en/stable/faq/#how-often-does-argo-cd-check-for-changes-to-my-git-or-helm-repository
